CSS/components/grids.css

```diff
@@ -0,0 +1,17 @@
+.grid-wrap-2-2{
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-rows: 1fr 1fr ;
+    grid-template-columns:  1fr 1fr;
+}
+.grid-wrap-3-3{
+    display: grid;
```

Comment:
You can take out display grid make that as utility class and use it

Reply:
created and  Added the util class for display: grid